### PR TITLE
Fix #363: pandas update to 2.0.0 changed line_terminator to lineterminator

### LINKED ISSUE #363: tests/xlsx_test.py::test_1 pykern/pkunit.py:741:_pkdlog ERROR converting xlsx to csv
```
____________________________________ test_1 ____________________________________
Traceback (most recent call last):
  File "/home/vagrant/src/radiasoft/pykern/tests/xlsx_test.py", line 22, in test_1
    for d in pykern.pkunit.case_dirs():
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 188, in case_dirs
    _compare(i, w)
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 173, in _compare
    file_eq(
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 277, in file_eq
    _FileEq(expect_path, *args, **kwargs)
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 517, in __init__
    self._set_expect_and_actual()
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 667, in _set_expect_and_actual
    if self._expect_csv():
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 622, in _expect_csv
    self._actual_xlsx()
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 529, in _actual_xlsx
    self._actual_xlsx_to_csv(b, s)
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkunit.py", line 550, in _actual_xlsx_to_csv
    p.to_csv(
TypeError: to_csv() got an unexpected keyword argument 'line_terminator'
```

### PR DESCRIPTION
**Pull request by e-carlin**

### CODE REVIEW

**Comment by e-carlin** [comment]
Updating python3 image to py3.9 then will re-run the tests

File: pykern/pkunit.py [hunk 1]
```diff
@@ -551,7 +551,7 @@ def _actual_xlsx_to_csv(self, actual_xlsx, sheet):
             str(self._actual_path),
             encoding="utf-8",
             index=False,
-            line_terminator="\r\n",
+            lineterminator="\r\n",
```
Comment (robnagler) [inline pykern/pkunit.py:554]: this needs to be conditional:
`TypeError: NDFrame.to_csv() got an unexpected keyword argument 'lineterminator'`

Comment (e-carlin) [reply pykern/pkunit.py:554]: You mean we should support both versions of pandas?

Comment (robnagler) [reply pykern/pkunit.py:554]: or force the version of pandas in pykern/setup.py and wherever else it is

**Pull request merged**